src/modules/IntegrationTracking.jl

```diff
@@ -41,8 +41,8 @@ end
 @def_integrator_struct(DriftKick)
 
 module IntegrationTracking
-using ..GTPSA, ..BeamTracking, ..StaticArrays, ..KernelAbstractions
-using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, @makekernel, Coords
+using ..GTPSA, ..BeamTracking, ..StaticArrays, ..KernelAbstractions, ..SIMD, ..SIMDMathFunctions
+using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, State_Alive, State_Lost, @makekernel, Coords
```

Comment:
```suggestion
using ..BeamTracking: XI, PXI, YI, PYI, ZI, PZI, Q0, QX, QY, QZ, STATE_ALIVE, STATE_LOST, @makekernel, Coords
```

Same for this file, all constants `State_alive` -> `STATE_ALIVE` and likewise for `State_lost`